modules/fts/pages/fts--advanced-default-analyzer.adoc

```diff
@@ -0,0 +1,4 @@
+= Default Analyzer
+The default analyzer to be used for this bucket.
```

Comment:
Don't use the word bucket here. 

The default analyzer is the one applicable to all the text fields across type mappings unless explicitly overridden.